### DISCUSSION TITLE
Joomla Error Screen Request for a change

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#39230] [General]
This is the Joomla 500 Error Screen:
![grafik](https://user-images.githubusercontent.com/828371/202434434-9cb55717-aad9-49ee-88b9-0a4a5660cd4b.png)

### What needs to be fixed
The Error screen should be less "bold" in my opinion and mention Joomla less.

### Why this should be fixed
Today a hoster of my client had an issue and the 500 Error page appeared. It advertises Joomla in a bad way, even if the website is not configured wrong, it looks like Joomla is the Problem. Also, the design does not look very modern, I would simplify it a lot.

### How would you fix it
I would propose a different design of the 500er Error page if there is agreement on the issue.

### Side Effects expected
no.

**Comment by brianteeman** [comment]
The biggest problem with that page is the help me resolve this button and what it actually says when you click on it. Everything else is very low priority for me.

**Reply by coolcat-creations** [reply]
So you don't see it as negative marketing to promote Joomla as the problem there?

**Reply by brianteeman** [reply]
No I dont. Its fairly standard. And reminds me of this https://www.theregister.com/2006/03/24/tuttle_centos/

**Reply by coolcat-creations** [reply]
Actually your link proves why it should be changed to something different because non-technical people do not understand. And that page / screen is public and shown to non experienced audience.

**Reply by Milo-W** [reply]
I don't know if an even more awesome feature would be to actually list some 5-7 links leading to top problems associated with 500, that would be totally pro user. Wonder what happens now if you click the main CTA though. Non experienced people would probably not be able to tackle it anyway ; )

**Reply by brianteeman** [reply]
That is basically what you get now although it's useless to a site visitor

**Reply by Milo-W** [reply]
Yeah but to an administrator / developer? Or do we not add anything because they will find out the solution anyway?

**Reply by coolcat-creations** [reply]
I would simply write: This site is down due to a Server or Configuration Error, please contact your Administrator. And make it just a bit less ugly.

**Reply by crystalenka** [reply]
I wonder if there would be a way to check cookies on the page to see if there was an admin user logged in at the time of error, and if so to show more detailed info.

**Reply by bembelimen** [reply]
> I wonder if there would be a way to check cookies on the page to see if there was an admin user logged in at the time of error, and if so to show more detailed info.

If you enable debug mode, you get more information.

**Reply by crystalenka** [reply]
@bembelimen yes of course but it's not necessarily the best thing to show all users 

Plus if it happens in the back end, or if the user is not technical enough to get into their config file to enable debug manually, they may not be able to turn it on without help (which the original error screen does not elaborate on)

**Comment by alikon** [comment]
hard to solve
https://en.wikipedia.org/wiki/Screen_of_death#White

**Comment by crystalenka** [comment]
I think there are a few aspects to this that could be improved. 

1. **The default design.** It's scary! People are already going to be panicking that something broke, whether they're a visitor to the site or the admin. 
2. **The error message(s).** It's not helpful, it's not really actionable, people don't know what to do next. 
3. **The lack of distinction between user roles.** Site admin needs more info on how to fix it. Site visitors just need to know that something broke and how to inform the site admin. (Or, ideally, that the site admin has already been notified.) And then they should be given next steps if possible (go to the home page or something). 

How exactly all of that would work is unclear but I definitely agree this needs to be improved in J5 at the latest.
